alf/algorithms/rl_algorithm.py

```diff
@@ -215,7 +215,7 @@ def __init__(self,
             replay_buffer_length = adjust_replay_buffer_length(
                 config, self._num_earliest_frames_ignored)
 
-            if config.whole_replay_buffer_training:
+            if config.whole_replay_buffer_training and config.clear_replay_buffer:
```

Comment:
If config.clear_replay_buffer=True, we won't care about the initial_collect_steps?